src/compiler/ast/module/SourceFile.ts

```diff
@@ -748,6 +747,22 @@ export class SourceFile extends SourceFileBase<ts.SourceFile> {
         return this;
     }
 
+    /**
+     * Removes all unused declarations like interfaces, classes, enums, functions, variables, parameters,
+     * methods, properties, imports, etc. from this file.
+     *
+     * Tip: Call this method twice in order to obtain optimal results. Sometimes there nodes that are only referenced
```

Comment:
It's good to tell people about how calling this multiple times might be useful, but as pointed out before, saying to call it twice is not necessarily sufficient.